catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroSeekBarColorBrickTest.java

```diff
@@ -0,0 +1,144 @@
+import org.catrobat.catroid.content.bricks.PhiroRGBLightBrick;
+import org.catrobat.catroid.ui.SpriteActivity;
+import org.catrobat.catroid.uiespresso.content.brick.utils.BrickTestUtils;
+import org.catrobat.catroid.uiespresso.content.brick.utils.CustomSwipeAction;
+import org.catrobat.catroid.uiespresso.testsuites.Cat;
+import org.catrobat.catroid.uiespresso.testsuites.Level;
+import org.catrobat.catroid.uiespresso.util.rules.BaseActivityInstrumentationRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+import static org.junit.runners.Parameterized.Parameter;
+import static org.junit.runners.Parameterized.Parameters;
+
+@Category({Cat.AppUi.class, Level.Smoke.class})
+@RunWith(Parameterized.class)
+public class PhiroSeekBarColorBrickTest {
+
+	private static final Integer MIN_COLOR_VALUE = 0;
+	private static final Integer MAX_COLOR_VALUE = 255;
+	private static final Integer INIT_COLOR_VALUE = 128;
+
+	private static Integer whenBrickPosition = 0;
+	private static Integer phiroRGBLightBrickPosition = 1;
+
+	@Rule
+	public BaseActivityInstrumentationRule<SpriteActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(SpriteActivity.class, SpriteActivity.EXTRA_FRAGMENT_POSITION, SpriteActivity.FRAGMENT_SCRIPTS);
+
+	@Parameters(name = "{3}-Test")
+	public static Iterable<Object[]> data() {
+		return Arrays.asList(new Object[][] {
+				{R.id.color_rgb_seekbar_red,
+						R.id.rgb_red_value,
+						R.id.brick_phiro_rgb_led_action_red_edit_text,
+						"SeekbarRed"},
+				{R.id.color_rgb_seekbar_green,
+						R.id.rgb_green_value,
+						R.id.brick_phiro_rgb_led_action_green_edit_text,
+						"SeekbarGreen"},
+				{R.id.color_rgb_seekbar_blue,
+						R.id.rgb_blue_value,
+						R.id.brick_phiro_rgb_led_action_blue_edit_text,
+						"SeekbarBlue"},
+		});
+	}
+
+	@Parameter
+	public @IdRes int colorRgbSeekbarId;
+
+	@Parameter(1)
+	public @IdRes int rgbValueId;
+
+	@Parameter(2)
+	public @IdRes int brickActionEditTextId;
+
+	@Parameter(3)
+	public String testName;
+
+	@Before
+	public void setUp() throws Exception {
+		Script script = BrickTestUtils.createProjectAndGetStartScript("PhiroSeekBarColorBrickTest");
+		script.addBrick(new PhiroRGBLightBrick(PhiroRGBLightBrick.Eye.BOTH, INIT_COLOR_VALUE, INIT_COLOR_VALUE, INIT_COLOR_VALUE));
+
+		baseActivityTestRule.launchActivity(null);
```

Comment:
please call baseActivityTestRule.launchActivity()
if you give it null, it ignores the fragment position given in the rules constructor.